api/ImageData.json

```diff
@@ -77,10 +77,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "6.1"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "7"
```

Comment:
Does the iOS version also need to be updated?

Reply:
Just the desktop data; Safari iOS 6.1 is the same WebKit version as Safari iOS 6.

Reply:
I see. A bit of a complicated version mapping :)